File: 브론즈 2 (5문제)/1297_TV 크기.py
# 참고 링크 : // 참고 링크 : https://moon-gd.tistory.com/27
# TV 정보 입력 받기
Diagonal, Height, Width = map(int, input().split())

# 피타고라스 법칙을 활용해서 비율 계산
ratio = (Diagonal ** 2 / (Height ** 2 + Width ** 2)) ** 0.5

# 계산된 비율에 맞춰서 높이, 너비 순으로 출력
print(int(ratio * Height), int(ratio * Width))

File: 브론즈 2 (5문제)/1297_TV 크기.c
// 참고 링크 : https://moon-gd.tistory.com/27
#define _CRT_SECURE_NO_WARNINGS
#include <stdio.h>
#include <math.h>

int main() {
	double diagonal, height, width;

	// 입력 받기
	scanf("%lf %lf %lf", &diagonal, &height, &width);

	// 비율
	double ratio = diagonal / pow((height * height + width * width), 0.5);

	// 비율을 대입해서 높이, 너비 계산
	printf("%d %d", (int)(ratio * height), (int)(ratio * width));

	return 0;
}